src/applications/vaos/components/NoValidVAFacilities.jsx

```diff
@@ -30,49 +28,23 @@ export default function NoValidVAFacilities({ formContext }) {
         content={
           <>
             <p>
-              We’re sorry. This facility doesn’t allow online appointments for
-              this type of care. Please call the medical center for more
-              information.
+              We’re sorry. This medical center and associated clinics don’t
+              allow online appointments for this type of care. Please call the
+              medical center for more information.
             </p>
             {parentDetails ? (
               <div className="vads-u-padding-left--2 vads-u-border-left--4px vads-u-border-color--primary">
-                <span className="vads-u-font-weight--bold">
-                  {parentDetails?.name}
-                </span>
-                <br />
-                <span>{address?.address1}</span>
-                <br />
-                {!!address?.address2 && (
-                  <>
-                    <span>{address?.address2}</span>
-                    <br />
-                  </>
-                )}
-                <span>
-                  {address?.city}, {address?.state} {address?.zip}
-                </span>
-                <br />
-                <FacilityDirectionsLink location={parentDetails} />
-                <div className="vads-u-display--flex vads-u-margin-top--2">
-                  <FacilityHours location={parentDetails} />
-                </div>
-                <p>
-                  <span className="vads-u-font-weight--bold">Main phone: </span>
-                  <a href={`tel:${phone?.main?.replace('-', '')}`}>
-                    {phone?.main}
-                  </a>
-                </p>
-                {!!phone?.mentalHealthClinic && (
```

Comment:
Similarly, FHIR doesn't really have different types of phone numbers, so removing this for now